README.rst

```diff
@@ -217,6 +219,10 @@ Building libmpv
 
 You can enable building libmpv by enabling the configure option::
 
+    # if using meson (default)
+    printf "%s\n" -Dlibmpv-shared=enabled > mpv_options
+
+    # if using waf
```

Comment:
I would just remove this entirely. Since waf is deprecated the idea is to not encourage it in any form.